src/auth.c

```diff
@@ -1716,13 +1716,16 @@ int pr_auth_chroot(const char *path) {
 
   tz = pr_env_get(session.pool, "TZ"); 
   if (tz == NULL) {
+    char tzbuf[32];
+    long off = (timezone / 60 / 60) - daylight;
+    snprintf(tzbuf, sizeof(tzbuf), "%s%ld", daylight ? tzname[1] : tzname[0], off);
```

Comment:
Ah, OK.  What do you think of having automatically setting the TZ value, if not found in the environment, to ":/etc/localtime"?  Per the tzset(3) and tzfile(5) man pages, that is the default assumption made by functions like ctime(3), if no TZ environment variable is found.